pkg/options/options.go

```diff
@@ -36,6 +36,7 @@ type CLI struct {
 		DartSymbol         upload.DartSymbolOptions      `cmd:"" help:"Process and upload symbol files for Flutter" name:"dart"`
 		ReactNativeAndroid upload.ReactNativeAndroid     `cmd:"" help:"Upload source maps for React Native Android"`
 		ReactNativeIos     upload.ReactNativeIos         `cmd:"" help:"Upload source maps for React Native iOS"`
+		Js                 upload.JsOptions              `cmd:"" help:"Upload source maps for the web"`
```

Comment:
We'll try and make this generic:

```suggestion
		Js                 upload.JsOptions              `cmd:"" help:"Upload JavaScript source maps"`
```